x/auth/tx/builder_test.go

```diff
@@ -1,7 +1,11 @@
 package tx
 
 import (
+	"github.com/cosmos/cosmos-sdk/client"
+	testutil2 "github.com/cosmos/cosmos-sdk/testutil"
+	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
 	"testing"
+	"time"
```

Comment:
Lint is whining because of import formating

lets just run `make lint-fix` to resolve